src/main/java/user/acceptconnect/AcceptConnectionUserResponseModel.java

```diff
@@ -5,6 +5,15 @@
 
 import java.util.ArrayList;
 
+/** A response model for the acceptconnect use case specificcally for users that frames the output
```

Comment:
It might be better if you do the java doc before the constructor (line 27), it'll automatically generate param which makes marking easier for the TA. 

Although we don't need constructor documentation for many classes, this one could be a special case, because the constructor does most of the job for the whole class, so we can just put the class doc before the constructor.

Reply:
When I was in class I asked the TA about constructor documentation and he said that we only comment constructors when they are overloaded, so I don't think he's looking for the param annotation about the constructor. I'm actually toeing the line of saying too much in the class descriptions when mentioning what it takes in / holds, usually that isn't really needed information. So I think here it's fine to leave it this way.

Reply:
I did hear the conversation, i was right next to you LOL, but since you are adding comment for the whole class anyways and you are literally going through every parameter to explain what they do, so i think it'll be much easier if you do it for the constructor instead

Reply:
I did hear the conversation, i was right next to you lol, but since you are adding comment for the whole class anyways and you are literally going through every parameter to explain what they do, so i think it'll be much easier for the grading purpose if you do it for the constructor instead. And i think the overloading thing is a general rule, there's definitely flexibility when you practice it. But anyways i don't think it'll be a big deal if you want to do it your way, i'm just making a suggestion